src/main/scala/cromwell/binding/package.scala

```diff
@@ -25,7 +25,7 @@ package object binding {
   type FullyQualifiedName = String
   type LocallyQualifiedName = String
   type CallInputs = Map[String, WdlValue]
-  case class CallOutput(wdlValue: WdlValue, hash: SymbolHash)
+  case class CallOutput(wdlValue: WdlValue, hash: Option[SymbolHash])
```

Comment:
We do it all the time:

    sealed trait CallOutput { 
      def wdlValue: WdlValue
    }
    final case class UnhashedCallOutput(wdlValue: WdlValue) extends CallOutput
    final case class HashedCallOutput(wdlValue: WdlValue, symbolHash: SymbolHash) extends CallOutput

What I meant was in this case I wasn't interested in another round of the ever present "I don't see value in that" argument